requirements.txt

```diff
@@ -1,3 +1,4 @@
+anndata==0.11.1
```

Comment:
👍

Reply:
the problem is that in order to use `0.11.1` we would need to drop support for python 3.8 (latest supported versions are 0.10.9, 0.11.0rc1, 0.11.0rc2) .. would be an argument for moving this module out of alphabase